index.js

```diff
@@ -373,11 +373,19 @@ module.exports = function (config, deps) {
         .set(common.SESSION_TOKEN_HEADER, user.getUserToken())
         .end(
         function (err, res) {
+          console.log("err", err, "res", res);
```

Comment:
Arrgh, I had that in there for debugging - didn't mean to push it up, will remove.